views/homepage.erb

```diff
@@ -0,0 +1,11 @@
+
+<div class="title">
+
+<h1 style="text-align:center"> Welcome to peep!</h1>
+</div>
+
+<form style="text-align:center">
+    <button type="submit" class="btn btn-primary btn-lg btn-block" formaction="http://localhost:9292/messages">view peeps!</button>
```

Comment:
here you've included `localhost` in your formaction, which means it'll only work when the client and server are on the same machine (as is the case when you're developing the app). If you were to deploy it to the cloud, this would break.